pages/wikilist/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import WikilistUI from './components/WikiListUI';
+
+const Wikilist: React.FC = () => {
+  return (
+    <div>
+      <WikilistUI />
+    </div>
+  );
```

Comment:
```suggestion
return <WikilistUI />;
```
`<div>`가 필요해서 사용한 것이 아니라면, 위에 제안처럼 바꿀 수 있을 것 같아요 ㅎㅎ